flutter-engine/src/plugins/textinput.rs

```diff
@@ -275,8 +275,8 @@ impl Plugin for TextInputPlugin {
     fn handle(&mut self, msg: &PlatformMessage, _: Arc<FlutterEngineInner>, _: &mut glfw::Window) {
         let decoded = self.channel.decode_method_call(msg);
 
-        info!("textinput mehod {:?}", decoded.method);
```

Comment:
These two log may have been mistakenly committed. If you don't need it, we could delete these.